ios/Podfile

```diff
@@ -79,7 +83,7 @@ post_install do |installer|
     use_native_modules![:reactNativePath],
     :mac_catalyst_enabled => false
   )
-  __apply_Xcode_12_5_M1_post_install_workaround(installer)
+#   __apply_Xcode_12_5_M1_post_install_workaround(installer)
```

Comment:
Is this no longer necessary?

Reply:
https://github.com/facebook/react-native/commit/0ab8b40fd64ad86b4598293faa0b77e71fc9d349